subsys/bluetooth/host/mesh/friend.c

```diff
@@ -462,6 +467,7 @@ int bt_mesh_friend_sub_add(struct bt_mesh_net_rx *rx,
 		return 0;
 	}
 
+	frnd->pending_req = 1;
```

Comment:
maybe it would be good to add helper for this two calls? Just to make sure that pendfing_req is always set when needed?

Reply:
Sure, not a bad idea. I've incorporated this into the latest version which I just uploaded.